virtcontainers/qemu.go

```diff
@@ -1567,6 +1587,32 @@ func (q *qemu) resizeVCPUs(reqVCPUs uint32) (currentVCPUs uint32, newVCPUs uint3
 	return currentVCPUs, newVCPUs, nil
 }
 
+func (q *qemu) updateMemoryBalloon(sizeMB uint32) error {
+
+	if q.state.HasVFIODevice {
+		// When VFIO devices are used we dont want to get pages back to
+		// the host. Lets increase the balloon to a large amount of
```

Comment:
@jcvenegas I havent looked a whole lot into memory balooning, but basic question here...if we dont want to give pages to the host, then shouldnt we be deflating the balloon here, so that no memory is returned to the host instead of increasing the baloon size?

Reply:
@amshinde increase the balloon size will increase the amount of memory available in the guest.  If we reduce it the pages will be given back to the host.  So ideally is never  get pages back to the host when we use VFIO.

Reply:
Dug a little bit into the actual qmp command
https://github.com/coreos/qemu/blob/master/qmp-commands.hx#L1830

Looks like this command actually sets the size of the VM and not the balloon!
So we need to update the docomentation here: 
https://github.com/intel/govmm/blob/78d079db6d1f3e32e3ed7578a54baa6257b058a7/qemu/qmp.go#L1353

Also, the comments here, also need to be updated to mention, that we are actually deflating the balloon and not inflating it.

Reply:
/cc @markdryan.